src/app/components/splash-screen/splash-screen.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit, OnDestroy , ViewEncapsulation } from '@angular/core';
+import { UtilityService } from 'src/app/services/utility.service';
+import {DEFAULT_IMAGE_TEXT} from "src/constants/constant";
+
+@Component({
+  selector: 'app-splash-screen',
+  templateUrl: './splash-screen.component.html',
+  styleUrls: ['./splash-screen.component.scss'],
+  encapsulation: ViewEncapsulation.None
+})
+export class SplashScreenComponent implements OnInit , OnDestroy {
+  showSplash: boolean;
+  imgText= DEFAULT_IMAGE_TEXT;
+  timeOutID: number;
+  iconImageUrl = "../../../assets/icons/logo.svg";
+
+  constructor(private _utilityService: UtilityService) {}
```

Comment:
the convention to use underscore `_` is to indicate that we want to ignore the use of a parameter, not as a hint that the field is private. the TS compiler will handle that perfectly.

```suggestion
  constructor(private utilityService: UtilityService) {}
```